data/schema/schema_template.yaml

```diff
@@ -16,24 +16,25 @@ endpoint_url: URL|URL[] # use and array of URLs if multiple files have been down
 derived_from: string[] # OPTIONAL - list of indicator keys this indicator derives from
 framework_group: string # key for the framework group that contains this indicator (from data/aux/framework.json)
 is_experimental: boolean # whether this indicator is experimental
-order: [year, nuts_id, nuts_year_spec, value.id] # USE AS IS: it specifies the order of the exported indicator fields
+order: [year, <region_type>_id, <region_type>_year_spec, value.id] # USE AS IS: it specifies the order of the exported indicator fields
```

Comment:
Is the comment string still value or does `<region_type>` need to be replaced with nuts/leps etc.?

Reply:
right, I'm removing `USE AS IS` and clarifying